airflow/api_fastapi/core_api/routes/public/job.py

```diff
@@ -16,10 +16,11 @@
 # under the License.
 from __future__ import annotations
 
+from typing import Annotated
+
 from fastapi import Depends, status
 from sqlalchemy import select
 from sqlalchemy.orm import Session
-from typing_extensions import Annotated
```

Comment:
Not sure is it use `typing_extensions` or `typing`. pre-commit failing. if it is `typing_extensions` is required may be we have to add ignore?

Reply:
This is right, it should come from Annotated.